jetty/src/test/scala/org/http4s/server/jetty/JettyServerSuite.scala

```diff
@@ -52,25 +52,25 @@ class JettyServerSuite extends Http4sSuite {
             IO.never
 
           case GET -> Root / "slow" =>
-            implicitly[Timer[IO]].sleep(50.millis) *> Ok("slow")
+            Temporal[IO].sleep(50.millis) *> Ok("slow")
         },
         "/"
       )
       .resource
+  }
 
   def jettyServer: FunFixture[Server] =
-    ResourceFixture[Server](serverR)
+    ResourceFixture[Server](Dispatcher[IO].flatMap(serverR))
 
   def get(server: Server, path: String): IO[String] =
-    testBlocker.blockOn(
-      IO(
-        Source
-          .fromURL(new URL(s"http://127.0.0.1:${server.address.getPort}$path"))
-          .getLines()
-          .mkString))
+    IO(
```

Comment:
This blocks, and unlike most of our blocking calls we've been porting, fs2 isn't going to handle it.

```suggestion
    IO.blocking(
```

Reply:
done